os-config/environment-test-ids.py

```diff
@@ -27,3 +27,11 @@
     ENV_ID_TRENCHBOOT: "trenchboot",
     ENV_ID_ESXI: "ESXi",
 }
+
+ENV_ID_OS_FRIENDLY_NAMES = {
```

Comment:
I think we should drop `OS` from name and include strings for firmware, even though they are rarely used right now.